server/src/main/resources/cloudformation/files/util/create-nodejs.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get update
```

Comment:
just a question: do you only want to update or maybe upgrade as well?
if not forget the question

Reply:
The reason why you run update here is to Download the "Package Database" for node js that has been added using the command above the update command.

There is no intention to run a full system upgrade (At least there was none when i wrote this script for the express model) We just want to fetch the latest version of the database to install the latest version of nodejs